src/pages/designing/index.mdx

```diff
@@ -88,23 +133,34 @@ expressive styles, check out
 
 </InlineNotification>
 
-## Accessibility
+### The 2x Grid
+
+Carbon for IBM.com inherits all the basic elements (grid, colors, typography, iconography) from the Carbon Design System. If you’re new to Carbon for IBM.com, the 2x Grid is a good place to start. The 2x Grid is the geometric foundation of all the visual elements of IBM Design and forms the basis for all designs at IBM and is therefore essential to understand.
+
+You can learn about the fundamentals of [using the 2x Grid](<[https://carbondesignsystem.com/guidelines/2x-grid/overview/](https://carbondesignsystem.com/guidelines/2x-grid/overview/)>) on the Carbon website.
```

Comment:
This link is breaking, maybe due to the PR branch but I'm not sure. I also think it might be good to use the old school syntax so we can have the link open in new tab: 

`You can learn about the fundamentals of <a href="https://carbondesignsystem.com/guidelines/2x-grid/overview/" target="_blank">using the 2x Grid</a> on the Carbon website.`